### PR TITLE
Moving AB test tracking to first omniture call

### PR DESCRIPTION
**Pull request by dominickendrick**
This PR:
- Adds you previous page ab test participations into the current page omniture tracking
- Fires a second Omniture call if you are added or removed from a test
- Does not fire a second call if your test allocation does not change

Depends on  #11789
## QA
- [ ] With no current test, if I'm not allocated to any new test, a 2nd tracking call shouldn't be fired
- [ ] With some current tests, if I'm not allocated to any new test, a 2nd tracking call shouldn't be fired
- [ ] With no current tests, if I am allocated to any new test, a 2nd tracking call should fire
- [ ] With some current tests, if I am allocated to any new test, a 2nd tracking call should fire
- [ ] With some current tests, if I am removed from a test, a second tracking call should fire

### CODE REVIEW

**Comment by dominickendrick** [comment]
@sndrs @desbo 
@ScottPainterGNM it would be good to verify the test plan with you!

File: common/app/templates/inlineJS/blocking/analytics.scala.js [hunk 1]
```diff
@@ -130,6 +130,43 @@ try {
...
+            // If it does, we fire a second tracking call in modules/analytics/omniture.js
+            guardian.config.abTestsParticipations = abTestsParticipations;
+        } catch (e) { }
+
+        function makeOmnitureABTag(currentParticipations) {
+            var participations = JSON.parse(currentParticipations);
+            var tag = [];
+
+            for (var key in participations.value) {
+                tag.push(['AB', key, participations.value[key].variant].join(' | '));
```
Comment (sndrs) [inline common/app/templates/inlineJS/blocking/analytics.scala.js]: might want `.hasOwnProperty(key)` check in these?

Comment (desbo) [inline common/app/templates/inlineJS/blocking/analytics.scala.js]: the result of json.parse isn't part of any prototype chain so we don't need any `hasOwnProperty` check.
screen shot 2016-02-04 at 10 59 11

Comment (sndrs) [inline common/app/templates/inlineJS/blocking/analytics.scala.js]: very true, it's probably overkill. ignore me :boom:

**Comment by johnduffell** [comment]
this sounds like a great idea, but is this to :ship: ? or :closed_book: ? @dominickendrick

**Pull request closed without merging**